frontend/common/src/messages/commonMessages.ts

```diff
@@ -26,6 +26,11 @@ const messages = defineMessages({
     id: "ufSiRU",
     description: "Title displayed when an item cannot be found.",
   },
+  notAvailable: {
+    defaultMessage: "N/A",
+    id: "A2Wa7Z",
+    description: "Message displayed when specific information is not available",
```

Comment:
Should be Not Applicable - https://dictionary.cambridge.org/dictionary/english/n-a

Reply:
I swear English is my first language 😆 

Fixed in [update to accurately represent N/A description](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4293/commits/67094b59c27d9a66d7e421df32f094f8065ca21b)